### PR TITLE
#1332 hide inactive users

### PR DESCRIPTION
**Pull request by krzyk**
#1332 
* added hiding of users that didn't receive any awards during the last 90 days
* added text informing about that on the team page

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1679](https://github.com/zerocracy/farm/issues/1679) is now in scope, role is `REV`

File: src/main/java/com/zerocracy/tk/TkTeam.java [hunk 1]
```diff
@@ -75,7 +78,12 @@ public Response act(final Request req) throws IOException {
                             input -> sources.add(this.source(input)),
                             true
                         ),
-                        new Xocument(item).nodes("/people/person[mentor]")
+                        new Filtered<>(
+                            node -> !new Awards(
```
Comment (g4s8) [inline src/main/java/com/zerocracy/tk/TkTeam.java]: @krzyk it would be faster to cache this value in `people.xml` (like all other properties)

Comment (krzyk) [reply src/main/java/com/zerocracy/tk/TkTeam.java]: @g4s8 yeah, I created https://github.com/zerocracy/datum/issues/386 for that just now

Comment (g4s8) [reply src/main/java/com/zerocracy/tk/TkTeam.java]: @krzyk can you also add a `todo` here to implement it later in this repo?

**Comment by krzyk** [comment]
@g4s8 updated

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:zerocracy/farm#1679` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#1679](https://github.com/zerocracy/farm/issues/1679) is now out of scope

**Pull request merged**